.github/workflows/lint.yml

```diff
@@ -20,8 +20,10 @@ jobs:
           node-version: 10.x
       - name: Set up ESLint
         run: npm install --global eslint
+      - name: Set up ESLint html plugin
+        run: npm install eslint-plugin-html@latest --save-dev
```

Comment:
This step wouldn't be necessary, we could use the previous step. I'll take I look into it.